quickwit/rest-api-tests/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
```

Comment:
maaaaaaaaaan it seems everytime I wake up there is a different way to describe dependencies and environemnt in python.

This one is nice. It looks like rust :D